tests/raft_log_replication.py

```diff
@@ -196,191 +183,44 @@ async def _unittest_raft_log_replication() -> None:
-            ),
-        ),
-    ]
-
-    for index, new_entry in enumerate(new_entries):
-        request = sirius_cyber_corp.AppendEntries_1.Request(
-            term=raft_node_1._term,
-            prev_log_index=index + 1,  # index: 1, 2
-            prev_log_term=raft_node_1._log[index + 1].term,
-            log_entry=new_entry,
-        )
-        metadata = pycyphal.presentation.ServiceRequestMetadata(
-            client_node_id=42,
-            timestamp=time.time(),
-            priority=0,
-            transfer_id=0,
-        )
-        response = await raft_node_1._serve_append_entries(request, metadata)
-        assert response.success == True
-
-    await asyncio.sleep(TERM_TIMEOUT + 1)
-
-    assert len(raft_node_1._log) == 1 + 3
-    assert raft_node_1._log[0].term == 0
-    assert raft_node_1._log[0].entry.value == 0
-    assert raft_node_1._log[1].term == 4
-    assert raft_node_1._log[1].entry.name.value.tobytes().decode("utf-8") == "top_1"
-    assert raft_node_1._log[1].entry.value == 7
-    assert raft_node_1._log[2].term == 8
-    assert raft_node_1._log[2].entry.name.value.tobytes().decode("utf-8") == "top_2"
-    assert raft_node_1._log[2].entry.value == 11
-    assert raft_node_1._log[3].term == 9
-    assert raft_node_1._log[3].entry.name.value.tobytes().decode("utf-8") == "top_3"
-    assert raft_node_1._log[3].entry.value == 12
-
-    # check if the new entries are replicated in the follower nodes
-    assert len(raft_node_2._log) == 1 + 3
-    assert raft_node_2._log[0].term == 0
-    assert raft_node_2._log[0].entry.value == 0
-    assert raft_node_2._log[1].term == 4
-    assert raft_node_2._log[1].entry.name.value.tobytes().decode("utf-8") == "top_1"
-    assert raft_node_2._log[1].entry.value == 7
-    assert raft_node_2._log[2].term == 8
-    assert raft_node_2._log[2].entry.name.value.tobytes().decode("utf-8") == "top_2"
-    assert raft_node_2._log[2].entry.value == 11
-    assert raft_node_2._log[3].term == 9
-    assert raft_node_2._log[3].entry.name.value.tobytes().decode("utf-8") == "top_3"
-    assert raft_node_2._log[3].entry.value == 12
-    assert len(raft_node_3._log) == 1 + 3
-    assert raft_node_3._log[0].term == 0
-    assert raft_node_3._log[0].entry.value == 0
-    assert raft_node_3._log[1].term == 4
-    assert raft_node_3._log[1].entry.name.value.tobytes().decode("utf-8") == "top_1"
-    assert raft_node_3._log[1].entry.value == 7
-    assert raft_node_3._log[2].term == 8
-    assert raft_node_3._log[2].entry.name.value.tobytes().decode("utf-8") == "top_2"
-    assert raft_node_3._log[2].entry.value == 11
-    assert raft_node_3._log[3].term == 9
-    assert raft_node_3._log[3].entry.name.value.tobytes().decode("utf-8") == "top_3"
-    assert raft_node_3._log[3].entry.value == 12
-
-    _logger.info("================== TEST 4: Add an already existing log entry ==================")
+    _logger.info("================== TEST 2: Add an already existing log entry ==================")
```

Comment:
So you just remove a test if it's not passing, or what?

![image](https://github.com/user-attachments/assets/31be285c-48bb-434a-8d84-e44f75148df2)

Reply:
These tests are no longer relevant -> [answer](https://github.com/OpenCyphal-Garage/cyraft/pull/11#discussion_r1679520402)